pkg/cache/placeholder_manager_test.go

```diff
@@ -110,11 +112,12 @@ func createAndCheckPlaceholderCreate(mockedAPIProvider *client.MockedAPIProvider
 	err := placeholderMgr.createAppPlaceholders(app)
 	assert.NilError(t, err, "create app placeholders should be successful")
 	assert.Equal(t, len(createdPods), 30)
-	var priority *int32
 	for _, tg := range []string{"tg-test-group-1-app01-", "tg-test-group-2-app01-"} {
 		for i := 2; i <= 9; i++ {
 			podName := tg + strconv.Itoa(i)
-			assert.Equal(t, priority, createdPods[podName].Spec.Priority, "Pod should not have been created")
+			var priority *int32
```

Comment:
let's keep this declaration before line no. 115.

Reply:
Updated it. Thank you.